server.go

```diff
@@ -2607,35 +2607,44 @@ cleanup:
 // the required services and adds the discovered peers to the address manager.
 // Each seeder is contacted in a separate goroutine.
 func (s *server) querySeeders(ctx context.Context) {
-	// Add peers discovered through DNS to the address manager.
-	seeders := s.chainParams.Seeders()
-	for _, seeder := range seeders {
-		go func(seeder string) {
-			addrs, err := connmgr.SeedAddrs(ctx, seeder, dcrdDial,
-				connmgr.SeedFilterServices(defaultRequiredServices))
-			if err != nil {
-				srvrLog.Infof("seeder '%s' error: %v", seeder, err)
-				return
-			}
+	go func() {
```

Comment:
I don't see any reason for another goroutine and a wait group on all of them.

Why not just do this?

```Go
// Add peers discovered through DNS to the address manager.
for _, seeder := range seeders {
	go func(seeder string) {
		ctx, cancel := context.WithTimeout(ctx, time.Minute)
		defer cancel()

		// Existing code...
	}()
}
```